articles/reporting/enterprise-metrics-definitions.md

```diff
@@ -0,0 +1,80 @@
+| Undetermined Bounces     | `count_undetermined_bounce`            | Number of emails that bounced due to undetermined reasons. |
+| Clicks                   | `count_clicked`                        | Number of times that links were clicked across all emails. |
+| Raw Clicks               | `count_raw_clicked`                    | Number of unique link clicks from recipients. |
+| Unique Clicks            | `count_unique_clicked`                 | Number of emails that had at least one link clicked. May take longer to update. |
+| Unique Clicks            | `count_unique_clicked_approx`          | Number of emails that had at least one link clicked. Approximated with a 5% error threshold. |
+| Unique Confirmed Opens   | `count_unique_confirmed_opened`        | Number of emails that were displayed or had at a link clicked. May take longer to update. |
+| Unique Confirmed Opens   | `count_unique_confirmed_opened_approx` | Number of emails that were displayed or had at a link clicked. Approximated with a 5% error threshold. |
+| Rendered                 | `count_rendered`                       | Number of times emails were displayed. Applies only to HTML emails. |
+| Unique Renders           | `count_unique_rendered`                | Number of emails that were displayed at least once. Applies only to HTML emails. May take longer to update. |
+| Unique Renders           | `count_unique_rendered_approx`         | Number of emails that were displayed at least once. Applies only to HTML emails. Approximated with a 5% error threshold. |
+| Initial Renders          | `count_initial_rendered`               | Number of times emails were displayed, as recorded by top open pixels. Applies only to HTML emails. |
+| Unique Initial Rendered  | `count_unique_initial_rendered`        | Number of emails that were displayed at least once, as recorded by top open pixels. Applies only to HTML emails. May take longer to update. |
+| Unique Initial Rendered  | `count_unique_initial_rendered_approx` | Number of emails that were displayed at least once, as recorded by top open pixels. Applies only to HTML emails. Approximated with a 5% error threshold. |
+| Spam Complaints          | `count_spam_complaint`                 | Number of spam complaints received from mailbox providers. |
+| Unsubscribes             | `count_unsubscribe`                    | Number of times a recipient clicked a tagged unsubscribe link or used the list unsubscribe header. |
+
+## Report Metrics Definitions
+
+The reports section of your account uses [Metrics API](#header-api-metrics-enterprise-definitions)
+data to build graphs and reports to help understand your email activity.
+Most of the numbers found in these reports are taken straight from the API.
+Some of numbers in the reports are simple calculations involving 2 of the API metrics.
+Here are the definitions for each of these calculated metrics, including the formulas used to calculate them:
```

Comment:
This is so good!

Reply:
* Targeted:  is actually injected + rejected so it's the number of emails you attempted to send through SparkPost
* Injected: should this be "received or generated" by SparkPost because if you inject via SMTP we do not "generate" anything
* Rejected:  I think this should be "number of emails not accepted by SparkPost".  The word "generated" doesn't actually apply to SMTP traffic
* Policy Rejection: "number of emails that were not accepted by SparkPost due to policy reasons"
* Bounces: can we replace the word "except" with "not including"